tests/integration/records/records_numeric_database_fixture.py

```diff
@@ -126,14 +127,18 @@ def bring_up(self):
             raise NotImplementedError(f"Please teach me how to integration test {self.engine.name}")
         print(f"Creating: {create_tables}")
         for statement in create_tables:
-            self.engine.execute(statement)
+            with self.engine.connect() as connection:
+                with connection.begin():
+                    connection.exec_driver_sql(statement)
 
     def quote_schema_and_table(self, schema, table):
         return quote_schema_and_table(self.engine, schema, table)
 
     def drop_table_if_exists(self, schema, table):
         sql = f"DROP TABLE IF EXISTS {self.quote_schema_and_table(schema, table)}"
-        self.engine.execute(sql)
+        with self.engine.connect() as connection:
+            with connection.begin():
```

Comment:
are you sure you need to start a transaction to drop table?

Reply:
nope!

Reply:
so... can't say why but I guess we do need this begin